src/redux/features/plantings/plantingsSlice.ts

```diff
@@ -1,4 +1,5 @@
 import { createSlice, PayloadAction } from '@reduxjs/toolkit';
+import { requestUpdatePlantingCompleted } from './plantingsThunks';
```

Comment:
can we move this thunk into another file maybe `plantingsAsyncThunks`?
Only reason is because `./plantingsThunks` imports some code from this file, we don't want a reverse dependency as well.

Reply:
That's true! Will move it